### ISSUE TITLE
aria attributes do not match their roles

### ISSUE THREAD

**Issue by C-Berlin** [shopware/platform#8669]
### Shopware Version

6.6.10.3

### Affected area / extension

Platform(Default)

### Actual behaviour

When checking accessibility with Lighthouse, we received the following error (see attachment) ‘[aria-*] attributes do not match their roles’. The aria attributes in question are all located in the footer. 

<img width="1123" alt="Image" src="https://github.com/user-attachments/assets/3d5727ef-7685-43e4-953c-985ea6d95b11" />

### Expected behaviour

see attachment

### How to reproduce

see attachment

**Comment by Phil23** [comment]
Have you activated the `ACCESSIBILITY_TWEAKS` feature flag?

**Comment by Phil23** [comment]
This should have been fixed with 6.7 or using the feature flag. Closing due to no response.
